draft-ietf-rats-architecture.md

```diff
@@ -286,10 +286,9 @@ Relying Party:
 
 ## Hardware Watchdog
 
-One significant problem is malware that holds a device
-hostage and does not allow it to reboot to prevent updates to be
-applied.  This is a significant problem, because it allows a fleet
-of devices to be held hostage for ransom.
+One significant problem is malware that holds a device hostage and does not allow it to reboot to prevent updates from being applied.
+Remote attestation aids in the verification process that may occur locally as well as it is less likely that the policy or verification processes have also been compromised if hosted on a separate system.
+This is a significant problem, because it allows a fleet of devices to be held hostage for ransom.
```

Comment:
But, in the case, the Relying Party is the Watch Dog timer in the TPM/secure enclave itself, {{TCGarch}} section 43.3.
So the Attestation Results are returned to the PC, and provided to the enclave.

If the watch dog does not receive regular, and fresh, Attestation Results as to the systems' health, then it forces a reboot.